### DISCUSSION TITLE
After the demo package is installed, the admin cannot be accessed normally

### DISCUSSION THREAD

**Discussion by lixiang123456789** [pimcore/pimcore#17495] [General]
![0c57023db5fdc1faef3973a6b87ff9e](https://github.com/user-attachments/assets/88a1d276-0047-4742-b56b-643d408205da)

**Comment by kingjia90** [comment]
`local:8000`? that's usually for Django/Ruby on Rails/AngularJS, it should be just without the port
It seems that the local server has some misconfiguration or an atypical one, rather than demo having some issues.

Is it Apache or Nginx? If Apache, please copy the `.htaccess` file in https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/Apache_Configuration/ , it's not provided out of the box
